File: client/Assets/LuaFramework/Scripts/Network/Protocal.cs

namespace LuaFramework {
    public class Protocal {
        ///BUILD TABLE
        /// 其实这个表在lua里也有一份，这个是lua用来处理事件分发的。 在c#里图 方便就直接用了， 这四个的值再也不能更改
        public const int Connect = 101;     //连接服务器
        public const int Exception = 102;     //异常掉线
        public const int Disconnect = 103;     //正常断线
        public const int Message = 104; // 发送网络消息
        public const int MessageFailed = 105; // 发送消息失败
    }
}

File: client/Assets/LuaFramework/Lua/Common/protocal.lua
--Buildin Table
Protocal = {
	Connect		= 101,	--连接服务器
	Exception   = 102,	--异常掉线
	Disconnect  = 103,	--正常断线   
	Message		= 104,--接收消息
	MessageFailed = 105,	-- 发送消息失败

	----------上面这个四个不能更改，因为c#层也做了这个四个的定义--------
}


